DsEdaData.sql

```diff
@@ -0,0 +1,761 @@
+BEGIN TRANSACTION;
+CREATE TABLE "Account" (
+	sf_id VARCHAR(255) NOT NULL, 
+	"Name" VARCHAR(255), 
+	"RecordTypeId" VARCHAR(255), 
```

Comment:
Any reason for "" @dgamboni just wondering :)

Reply:
@jofsky I used the task `cci task run extract_dataset` which automatically created the file. I uploaded it without edits.